packages/web/src/components/gcds-signature/gcds-signature.tsx

```diff
@@ -95,18 +95,39 @@ export class GcdsSignature {
     }
   }
 
+  private get svgLabel() {
+    if (this.lang === 'en') {
+      return (
+        <gcds-sr-only tag="span" id="signature-title">
+          Government of Canada / <span lang="fr">Gouvernement du Canada</span>
+        </gcds-sr-only>
+      );
+    } else {
+      return (
+        <gcds-sr-only tag="span" id="signature-title">
```

Comment:
@ethanWallace seems like `<gcds-sr-only>` is repeated - wondering if the repetition can be minimized if the `<spans>`s were pulled into variables then using the ternary operator for the content between the tags?